Assets/MRTK/SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -466,7 +473,7 @@ private LinePointer GetControllerRay(Handedness handedness)
         /// <returns>true if not tracking valid hands and/or target, false otherwise</returns>
         private bool IsInvalidTracking()
         {
-            if (trackingTarget == null)
+            if (trackingTarget?.transform.parent == null)
```

Comment:
Since `trackingTarget` is a `GameObject`, we should use an explicit `trackingTarget == null` check here instead of `?.`

https://blogs.unity3d.com/2014/05/16/custom-operator-should-we-keep-it/
```suggestion
            if (trackingTarget == null || trackingTarget.transform.parent == null)
```